server/formats.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  FormatLookup,
+  Spectral,
+  isJSONSchema,
+  isJSONSchemaDraft2019_09, // eslint-disable-line
+  isJSONSchemaDraft4,
+  isJSONSchemaDraft6,
+  isJSONSchemaDraft7,
+  isJSONSchemaLoose,
+  isOpenApiv2,
+  isOpenApiv3,
+} from '@stoplight/spectral';
+
+/**
+ * Mapping of format ID to detector function that can determine if a given
+ * document is of that type.
+ */
+const allFormats: Array<[string, FormatLookup]> = [
```

Comment:
@P0lip Maybe should we export that collection out of Spectral to make integration a little bit more easier?

Reply:
@P0lip cf. https://github.com/stoplightio/spectral-action/blob/master/src/spectral.ts does the same kind of thing.

Reply:
Yeah, we could export them. I'll do it later today.